policy/cnspec_policy.proto

```diff
@@ -33,16 +56,60 @@ message PolicySpec {
   int64 created = 32;
   int64 modified = 33;
 }
+// ^^
+
+message PolicyRef {
+  enum Action {
+    UNKNOWN = 0;
+    ADD = 1;
+    DISABLE = 2;
+    MODIFY = 3;
+  }
+
+  string mrn = 1;
+  string uid = 2;
+
+  repeated cnquery.explorer.Property props = 35;
+  cnquery.explorer.Filters filters = 43;
+  cnquery.explorer.Impact.ScoringSystem scoring_system = 10;
+  // Action is used for all policy overrides
+  Action action = 41;
+
+  string graph_content_checksum = 38;
+  string graph_execution_checksum = 40;
+}
 
 /*
   Policy definition, re-usable or assigned to the user's environment
   - Contains all policies and overrides
   - Can limit to what it supports
 */
 message Policy {
+  // FIXME: DEPRECATED, remove in v9.0 vv
+  // This is replaced by the field "filters"
+  map<string,Mquery> v1_asset_filters = 7; // key == checksum, internal
+  repeated DeprecatedV6_PolicySpec v1_specs = 6;
+  // ^^
+
   string mrn = 1; // will be filled from UID on upload
+  string uid = 36; // uid is only used initially, to generate the MRN
   string name = 2;
   string version = 3;
+  string owner_mrn = 8;
+
+  repeated PolicySpec specs = 11;
+
+  bool is_public = 9;
+  string license = 21;
+  PolicyDocs docs = 41;
+
+  ScoringSystem scoring_system = 10;
+
+  repeated Author authors = 30;
+  int64 created = 32;
+  int64 modified = 33;
+  map<string,string> tags = 34;
+  repeated cnquery.explorer.Property props = 35;
```

Comment:
When we re-use the old proto buf id, are we sure we can load the old protobuf format?